applications/ortege.md

```diff
@@ -0,0 +1,267 @@
+
+In summary, while there is still work to be done, we are proud of the progress we have made so far and are excited about the potential of Ortege to revolutionize blockchain interoperability. 
+
+## Development Roadmap :nut_and_bolt: 
+ ### Overview 
+- **Total Estimated Duration:** 5 months 
+
+- **Full-Time Equivalent (FTE):**  4 FTE (Milestones will be worked on concurrently) 
+
+- **Total Costs:** $95,000 
+
+  
+
+### Milestone 1 Ink! Smart Contracts 
+
+We will begin by developing smart contracts in the ink! language. These contracts will form the backbone of our protocol, enabling the core functionality of cross-chain communication. We will leverage the design principles and specifications provided by Hyperlane to ensure our contracts are robust, secure, and efficient. This will involve creating contracts for managing the protocol's state, handling message routing, and facilitating the transfer of value across chains. 
+ 
+- **Estimated duration:** 4 months 
+- **FTE:**  2.5 
+- **Costs:** 60,000 USD 
+
+| Number | Deliverable | Specification | 
+| ------ | ----------- | ------------- | 
+| **0a.** | License | Apache 2.0 | 
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can query, call or post an interchain message. | 
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. | 
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. | 
+| **0e.** | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). | 
+| **1.** | Ink! Contract: MerkleTree | We will create an ink! MerkleTree implementation and will submit to OpenBrush for inclusion in their standards | 
+| **2.** | Ink! Contract: Interchain security module | The ISM contract will provide modular security for the Ortege system, accommodating use-case-specific security requirements. | 
+| **3.** | Ink! Contract: Mailbox | The Mailbox contract will facilitate the sending and receiving of messages between chains. | 
+| **4.** | Ink! Contract: Interchain Gas Paymaster | This contract will store exchange rates between gas tokens, thus ensuring enough gas for successful message execution. | 
+| **5.** | Ink! Contract: Chainlink ISM and Price Aggregator | These are the smart contracts that will allow dApp developers to query Chainlink’s prices and also cover the security of the queries | 
+| **6.** | Smart contract integration testing | All internal testing of the smart contracts and writing of unit tests | 
```

Comment:
How is this different from `0c`?

Reply:
One is unit tests, the other is integration testing